From 86be19a338a9fe1157a97c3bbed7fb63926f0db0 Mon Sep 17 00:00:00 2001
From: "Lopez Esparza, Ricardo" <ricardo.lopez.esparza@intel.com>
Date: Tue, 1 Mar 2016 18:12:40 -0600
Subject: [PATCH] OV8858: Fix BLC registers for sensor ov8858

The values of the BLC registers for CHT HR and CHT MRD are different.
This patch adds the register values needed for MRD.

Change-Id: I1e9ab3c71edf8df4ed3f1b4945613ca4c0125e58
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-22350
Signed-off-by: Lopez Esparza, Ricardo <ricardo.lopez.esparza@intel.com>
Reviewed-on: https://android.intel.com:443/476521
---
 drivers/external_drivers/camera/drivers/media/i2c/ov8858.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/ov8858.h b/drivers/external_drivers/camera/drivers/media/i2c/ov8858.h
index 444a79bb..2d84b8a 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/ov8858.h
+++ b/drivers/external_drivers/camera/drivers/media/i2c/ov8858.h
@@ -1253,6 +1253,10 @@ static const struct ov8858_reg ov8858_1640x926[] = {
 };
 
 static const struct ov8858_reg ov8858_BLC_MRD[] = {
+	{OV8858_8BIT, 0x4022, 0x0C}, /* Anchor left end */
+	{OV8858_8BIT, 0x4023, 0x60}, /* Anchor left end */
+	{OV8858_8BIT, 0x4024, 0x0F}, /* Anchor right start */
+	{OV8858_8BIT, 0x4026, 0x0F}, /* Anchor right end */
 	{OV8858_8BIT, 0x402E, 0x04}, /* Bottom black line start = 12 */
 	{OV8858_8BIT, 0x402F, 0x08}, /* Bottom black line number = 2 */
 	{OV8858_TOK_TERM, 0, 0}
-- 
1.9.1

